### ISSUE TITLE
Blank Page on 1.9.1

### ISSUE THREAD

**Issue by vroberttaken** [magento-hackathon/Magento-Two-factor-Authentication#12]
Hi,
I have problem with Magento-Two-factor-Authentication on magento 1.9.1. After I have installed the module an refreshed the cache and logout, because I get the standard error 'access is denied'  and than  I am trying to login after entering the username and pw I get a blank page and cannot acces the admin page. Than I also refreshed the session and still the same. A blank page 
How to fix that ?

**Comment by mzeis** [comment]
Hi,

did you enable `System > Configuration > Admin > Developer > Template Settings > Allow Symlinks`? Depending on how you installed the extension this might cause a blank page.

**Comment by avoelkl** [comment]
I can confirm it works with Magento 1.9.1.1 and Symlinks enabled.

**Comment by andrew-smart** [comment]
I can also confirm it works on 1.9.1 but i did get the exact "problem" described by @vroberttaken which was fixed as soon as i enabled symlinks as described by @mzeis

**Comment by dwaipayan** [comment]
Hi,
I have installed but it is showing this error in admin
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'admin_user_question' doesn't exist, query was: SELECT `main_table`.* FROM `admin_user_question` AS `main_table` WHERE (user_id = '1') ORDER BY question_id ASC
Why this is coming? What is the issue?

**Comment by avoelkl** [comment]
Did you clear the cache after installation? @dwaipayan

**Comment by dwaipayan** [comment]
Yes after clearing the cache when I logged out from admin and login again then showing this error
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'admin_user_question' doesn't exist, query was: SELECT main_table.* FROM admin_user_question AS main_table WHERE (user_id = '1') ORDER BY question_id ASC
Why the issue is happening I cannot able to track.What is the issue here?

**Comment by avoelkl** [comment]
@dwaipayan I think some of your upgrade scripts did not run as the `admin_user_question` table was added in the upgrade-0.1.2-0.1.3 setup script.

- Did you check the core_resource table if the module was added there in the right version? There should be an entry for "twofactorauth_setup" with version 0.1.3.
- Which version of Magento did you install this on?
- Are symlinks (System > Configuration > Admin > Developer > Template Settings > Allow Symlinks) enabled?
- Which branch of this module here are you using? Develop branch or master branch?

**Comment by dwaipayan** [comment]
Hello,
Thanks for your quick response.
I am using magento 1.9.1.0 community version.
No symlinks are not enabled here.
I am using Magento-Two-factor-Authentication-develop version 
Yes twofactorauth_setup" with version 0.1.3 in core_resource table present.

Still after refresh the cache the site is showing error.What is the issue please let me know.
Thanks,
dwaipayan
